site/glossary.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="urgen-lama">
+    <meta name="description" content="This is a webpage that has details about guitar and music production.">
+    <title>Title</title>
+    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+</head>
+<body>
+<div class="width-wrap">
+    <header id="top"> <!--Found me!-->
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section id="creating-a-glossary">
+
+        </section>
+    </main>
+
+    <footer>
+        <hr>
+        <blockquote>
+            "Every action has equal and opposite reaction"&mdash;
+            <cite><span class="author">Newton</span></cite>
+        </blockquote>
+        <p>
+            &copy; 2024 <a href="mailto:https://outlook.office.com/mail/#code=1.AScAnd1hnCk7bU-zsfIS3jwl2iC_mZE_oQdBhdwCEUeH70gnABknAA.AgABBAIAAABVrSpeuWamRam2jAF1XRQEAwDs_wUA9P9H1fC7QLMcv_Zxlg2ZVFyH6QTXgdhuOg9rIRtvdtA9QpC22uuQ3SLtHFMege0NgM7P_9ylqPsIgs_PL71NYtmL8kEVAgZBayoRdq0JlVlSHZ08pfbgx43XGVgAL7h3rCfETFOOkyiZ4gXG0onwZrc3lWXmp4wlMNpf7-sqJPIBNhEfEMpmgi25uFRgz9zS6Q_ZxboXmyy2MrEp7sxXWYvw43-qjIIrzwtJSq7aq4Dcz4SYnhwjsicZ1qzFyd7aJ4F5-aWdqvztaCNw4SvkGf7UEzIG_ATx5MYjJwNSmsYShMAAhyzu-YEUJolN_fWFrFGUk6encSybetwAUYwlocbprDwCebW69qg6j7_jjJ-XH8mp-5HfBEcj6rc2q0uo8w745WHiAPKDta9TfKJsEUQvvkR9ckuke7Pd5W0upuAoyXRsintRAT-HsZyD-87WTH4J8sIH7PXcKAuVV_ofuqpTV0CkjWL1l1-eqxGeygHY5QlEm6niIEFqFpxzZEuqy9BWOxmx2X9xWUVSz4pzTouVpqqOeAHxAljHMPMUKf5Xk3Su_VDsl-b2O1zjtcedIgXRnN8JRaqQxr1T3XZU5wz93p0Kv9xn2AbJIZtcicXNyK9OLTstTE341XEFDJlEvMfBwx5yVmXEB8fYx94rNh-M9ckJgJY5HLmoUHdkTdjjj56rilVXnO_ePQUDd-ZpI-XMxiMxCLx-inmE_iEuxR_4TTIfY2C46Dy_n8gVHtNhFwr4pyNHspCLS9BUFQqSyV0TOl5wD526tTh3_nWbu0GBZ0eKIm5UUD0onRJ6xgaRRixGx_gyw-gK5uq5c7WVKBj_RC7DY1N-FZMme3mxvJFMuylojIUMjCuyxoKZKTI9nHzJhtuMK6W0hqdfAGoyQiVMwPQ4Nw&client_info=eyJ1aWQiOiI3Yjk4YzAzMS1lYTcyLTRiNDYtOWJlYi0yYTBhYzM4MWFjMzIiLCJ1dGlkIjoiOWM2MWRkOWQtM2IyOS00ZjZkLWIzYjEtZjIxMmRlM2MyNWRhIn0&state=eyJpZCI6IjAxOTZmNGZmLTAwMzEtNzRmZi1iNTkzLWYzYjlhMDUwYWIxZSIsIm1ldGEiOnsiaW50ZXJhY3Rpb25UeXBlIjoicmVkaXJlY3QifX0%3d&session_state=003e9789-5c48-9da7-7417-158b8f05b363?subject=Message from CSD112 Website">Urgen</a>
```

Comment:
```suggestion
            &copy; 2024 <a href="mailto:anEmail@domain.com?subject=Message from CSD112 Website">Urgen</a>
```
Something like this was expected.